pkg/webhook/overcommit/handler.go

```diff
@@ -0,0 +1,206 @@
+/*
+Copyright 2022 The Arbiter Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package overcommit
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	v1 "k8s.io/api/admission/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	"github.com/kube-arbiter/arbiter/pkg/apis/v1alpha1"
+	"github.com/kube-arbiter/arbiter/pkg/generated/clientset/versioned"
+)
+
+var (
+	OvercommitName string
+)
+
+type PatchEntry struct {
+	Op    string      `json:"op"`
+	Path  string      `json:"path"`
+	Value interface{} `json:"value"`
+}
+
+func Handle(w http.ResponseWriter, r *http.Request) {
+	var body []byte
+
+	if r.Body != nil {
+		if data, err := io.ReadAll(r.Body); err == nil {
+			body = data
+		}
+	}
+
+	contentType := r.Header.Get("Content-Type")
+	if contentType != "application/json" {
+		klog.Errorf("Content-Type=%s, expert application/json", contentType)
+		return
+	}
+
```

Comment:
I think there should add some logic to judge if the namespace is matched.
`namespaceList` is defined, but can't find where to use it

Reply:
- user fill the field namespacesList of overcommit resource, svc (eg.tce apiserver) get the resource(overcommit) then to modify the label of namespace, to modify the namespaceSelector of mutatewebhook. so webhook needn't to that logic.
- configmap also suitable for the overcommit scene, but we choose the way of crd